docs/Early Access - Changes Affecting Extenders.md

```diff
@@ -11,3 +11,43 @@ Zowe Explorer's view IDs have been updated to improve the consistency of the for
 | Data Sets view | `zowe.explorer`     | `zowe.ds.explorer`   |
 | USS view       | `zowe.uss.explorer` | `zowe.uss.explorer`  |
 | Jobs view      | `zowe.jobs`         | `zowe.jobs.explorer` |
+
+## Migration of old settings to new standardized settings in Zowe Explorer
+
+### Overview of new standardized name settings
+
+The following table describes the changes from old to new standardized names for Zowe Explorer configurations:
```

Comment:
Maybe add some reasoning here like, "These changes were made to conform to VS Code's [configuration schema](https://code.visualstudio.com/api/references/contribution-points#Configuration-schema)".